### ISSUE TITLE
Client Fatal error after pairing

### ISSUE THREAD

**Issue by Jebsta** [gajosu/ha-whatsapp-web-rest-api#180]
### Description of the issue

Addon crashes after I paired my whatsapp.
Maybe because of outdated Whatsapp-web.js?
Bumped Verison in #178 

### Addon version

2.3.9

### Platform

Core 2024.9.2
Supervisor 2024.09.1
Operating System 13.1 

### Logs of the issue (if applicable)

```log
9/14/2024, 14:29:38 [error]: 	[Whatsapp] Client fatal error
Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'default')
    at __puppeteer_evaluation_script__:5:95
Error: Evaluation failed: TypeError: Cannot read properties of undefined (reading 'default')
  at __puppeteer_evaluation_script__:5:95
  at ExecutionContext._evaluateInternal (/app/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)
  at async ExecutionContext.evaluate (/app/node_modules/puppeteer/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)
  at async Client.initialize (/app/node_modules/whatsapp-web.js/src/Client.js:323:9) 
[14:29:38] WARNING: Halt add-on with exit code 1
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service base-addon-timezone: stopping
s6-rc: info: service base-addon-log-level: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service base-addon-timezone successfully stopped
s6-rc: info: service base-addon-log-level successfully stopped
s6-rc: info: service base-addon-banner: stopping
s6-rc: info: service base-addon-banner successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
```

**Comment by gajosu** [comment]
Hi! Check if it has been fixed in the latest version.

**Comment by Jebsta** [comment]
The build of 2.4.0 failed

**Comment by gajosu** [comment]
Fixed in v2.4.1
